manageiq-schema.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activerecord-id_regions", "~> 0.3.0"
   s.add_dependency "linux_admin", "~> 2.0"
   s.add_dependency "manageiq-password", "~> 0.3"
-  s.add_dependency "more_core_extensions", "~> 3.5"
+  s.add_dependency "more_core_extensions", "~> 4.0"
```

Comment:
Maybe this should be `">= 3.5", "< 5"`?  In #244 it sounds like we only really care that the minimum is 3.5.